webapp/locales/en.json

```diff
@@ -10,9 +10,9 @@
   },
   "admin": {
     "categories": {
-      "categoryName": "Name",
       "name": "Categories",
-      "postCount": "Posts"
+      "postCount": "Posts",
+      "categoryName": "Name"
```

Comment:
Is that to let Travis CI fail?

Reply:
Yes. I'll rebase-force the branch and then the tests should pass.